resources/lint/profiles/mercury.base-uri.test.ts

```diff
@@ -23,7 +22,6 @@ describe("base uri", () => {
       "http://a.ml/vocabularies/data#base-uri-matches-pattern";
     const TEMPLATE_RULE =
       "http://a.ml/vocabularies/data#camelcase-template-parameters";
-    const TITLE_RULE = "http://a.ml/vocabularies/data#title-matches-api-name";
```

Comment:
If this is being removed as it's unused, should it still be present and we're missing functionality?
If so, please create a new story to address it.

Reply:
@gkhunger-sf looks like you added this in #26.

Reply:
We never added this rule. I opened a PR and found that this rule is going away in the next api spec. I don't know how it ended up here. Thanks for removing it, Will.